janus_core/calculations/md.py

```diff
@@ -43,6 +44,7 @@
 from janus_core.processing.correlator import Correlation
 from janus_core.processing.post_process import compute_rdf, compute_vaf
 
+units = create_units("2014")
```

Comment:
While "consistent", it's slightly confusing to have `UNITS` and `units` simultaneously. Also, not sure what the difference between `create_units` (from ASE) and `_set_units` is at a glance.

Reply:
What does `set_info_units` mean? `get_unit_names`? `UNITS` -> `JANUS_UNITS` and/or `units` -> `ase_units` works, but should be done through qualified import (i.e. `import X as ...`)

Reply:
It's not just getting units, it's setting something like `struct.info["units"]`, so it is literally setting the units, to the info dictionary.

As in, `import UNITS as JANUS_UNITS`? Why not just rename `UNITS` in `calculations/base`, where it's defined?

Reply:
Ok, that answers the question. I'm happy with `_set_info_units`.

The reason I said that is mostly because `UNITS` is not a generally a bad name for import, it's only because we have a conflicting name from ASE that it's a problem.